### ISSUE TITLE
[Anti-adblock] NEW egy.best

### ISSUE THREAD

**Issue by mimo2004moh** [NanoAdblocker/NanoFilters#376]
### URLs where this issue occurs (Required)
https://www.egy.best

### Screenshots (Required)
first time open the site
![Capture1](https://user-images.githubusercontent.com/53051408/61584737-7da04e00-ab4d-11e9-8f36-9e3819ba9283.PNG)
after 3 click on the player anti-adblock appear
![Capture](https://user-images.githubusercontent.com/53051408/61584744-94df3b80-ab4d-11e9-888e-89eaa43fd3db.PNG)

### Screenshot of console (Required, press `F12` to open the console)
![Capture3](https://user-images.githubusercontent.com/53051408/61584787-e5569900-ab4d-11e9-8466-eda55450db57.PNG)

### Describe the issue (Optional if obvious)
anti-adblock appear after trying play the movie
when i disable nano adblocker and play the movie the site open 3 popups 
so the anti-adblock appear because the popups blocked by nano

#### Environment (Required)

- Operating System and Version: windows 10 x64
- Browser and Version: firefox 68.0.1
- Nano Adblocker Version: 1.0.0.120
- Nano Defender Version (Optional if you are not using it): 15.0.0154

#### Your filter lists (Required)
none

**Comment by mimo2004moh** [comment]
![Capture135](https://user-images.githubusercontent.com/53051408/61584830-14b9d580-ab4f-11e9-850b-5da31e8bb5d4.PNG)

**Comment by krystian3w** [comment]
duplicate of #375

**Comment by Hassene66** [comment]
same problem here

**Comment by krystian3w** [comment]
AdGuard solutions:

https://github.com/AdguardTeam/AdguardFilters/commit/3f2d20fa51c05f12559525676bcbc1103bd6a1aa

Maybe some scriplets possible translate to "uBO resources scriplets".

**Comment by jspenguin2017** [comment]
@krystian3w How's that a solution? They just commented out every filter that affect this domain.

**Comment by mimo2004moh** [comment]
i think the problem is in the site they upload their movies to 
this https://vidstream.top
try this movie link https://vidstream.top/v/HQBSdyQA0M/?vclid=f96d5c5ebc194c21277fabe173d03ddd4caf121e581b83791b46f385DbbbBPGxbBbIbBoFvWRYoXGAbBIBbBTPdbBbIbBAAAbrovprovRYpBvWYBIBbBlpoOBbIbBXjZepoFIuZtobFIoHyOqFbbBbw

**Comment by krystian3w** [comment]
Almost all have removed comments:

https://github.com/AdguardTeam/AdguardFilters/commit/0ffb29bf949e5c937d9a9fd109b1149adb8caa16

Only one newtwork has changed.

**Comment by Iaycomand** [comment]
sorry if I don't get it, is there any kind of solution yet? I still see the anti ad message by Russian server.

**Comment by llacb47** [comment]
AdGuard has scriplet solution, it cannot be implemented into uBO due to filter limitations.

**Comment by mhta26** [comment]
Is there a solution to this problem?

**Comment by krystian3w** [comment]
For my video's again Play - news links to test?

**Comment by krystian3w** [comment]
egybest.com,egybest.site,egy.best##^script:has-text(['removeCookie']):min-text-length(10000)

Maybe help on Firefox 57 and uBO 1.20+

**Comment by larcn** [comment]
any solution ?

**Comment by krystian3w** [comment]
Userscript?

**Comment by Iaycomand** [comment]
> Userscript?

What do you mean? can you ELI5 please?

**Comment by krystian3w** [comment]
`Violentmonkey` / `Tampermonkey` with copied AdGuard script filter.

**Comment by Iaycomand** [comment]
> `Violentmonkey` / `Tampermonkey` with copied AdGuard script filter.

Should I copy the green lines only from the script you provided earlier?

**Comment by krystian3w** [comment]
@Iaycomand:

```js
// ==UserScript==
// @name      AdGuard Egy Test
// @namespace GithubNanoAdGuard
// @match     *://*/*
// @grant     none
// @run-at    document-start
// ==/UserScript==
(function() {
    var b = window.setTimeout;
    window.setTimeout = function(a, c) {
        if (!/new Date\(\)[\s\S]*?WinMsgHtml|_AdBlock|WinMsgHtml\(decodeURIComponent/.test(a.toString())) return b(a, c)
    };
})();
(function() {
    var w_open = window.open,
        regex = /click\.php/;
    window.open = function(a, b) {
        if (typeof a !== 'string' || !regex.test(a)) {
            w_open(a, b);
        }
    };
})();
```

Maybe works enough fast.

**Comment by AbdelrahmanHafez** [comment]
@krystian3w Adding this as a user script on Tampermonkey doesn't seem to fix the problem for me.

**Comment by llacb47** [comment]
@AbdelrahmanHafez how are you getting to the site? When I try to visit it I get redirected to ads.

**Comment by AbdelrahmanHafez** [comment]
Try the following link: https://beta.egybest.cheap/episode/breaking-bad-season-5-ep-14/#download

**Comment by krystian3w** [comment]
> Adding this as a user script on Tampermonkey doesn't seem to fix the problem for me.

script maybe need add XHR privilege

`// @grant        GM_xmlhttpRequest` and no have converted:

`vidstream.top#%#AG_onLoad(function() { var el = document.querySelector('p > a.bigbutton:first-child:not(._reload)'); var el1 = document.querySelector('.video-js:not(._reload)'); if(el||el1) { $('html').unbind('click'); } });`,

**Comment by llacb47** [comment]
Doesn't work, I get redirected to `stremanp.com`, an adserver. Maybe it depends on location (there is a geolock)? I am in US. What country are you visiting from?

**Comment by krystian3w** [comment]
For me serve: 

![obraz](https://user-images.githubusercontent.com/35370833/70867882-c7fded80-1f7a-11ea-8475-9f62fb40e225.png)

**Comment by llacb47** [comment]
You are in Poland right @krystian3w ? So maybe only US/NA is redirected? Can you test @jspenguin2017

**Comment by jspenguin2017** [comment]
This one is probably going to be sticky, not worth the effort.

**Comment by jspenguin2017** [comment]
Also with this test link: https://github.com/NanoAdblocker/NanoFilters/issues/376#issuecomment-565837358
I get 404.

**Comment by AbdelrahmanHafez** [comment]
Ah, too bad. This is really popular in Egypt, removing ads would be a great improvement for users.

> This one is probably going to be sticky, not worth the effort.
